Configurations/10-main.conf

```diff
@@ -1260,6 +1260,32 @@ my %targets = (
         perlasm_scheme   => "elf",
     },
 
+# QNX
+    "qnx" => {
+        inherit_from     => [ "BASE_unix" ],
+        template         => 1,
+        cc               => "gcc",
+        cflags           => "-D__unix__ -DL_ENDIAN",
+        ex_libs          => add("-lsocket -lm"),
+        enable           => add("devcryptoeng"),
```

Comment:
I believe the devcrypto engine is being removed in 4.0, this probably shouldn't be added here.